docs/tools/erc725js/classes/ERC725.md

```diff
@@ -224,7 +224,7 @@ ERC725.decodePermissions('0x0000000000000000000000000000000000000000000000000000
 {
   CHANGEOWNER: false,
   CHANGEPERMISSIONS: false,
-  ADDPERMISSIONS: false,
+  ADDCONTROLLER: false,
```

Comment:
Good to have changed this too 👍 

@Hugoo are these pages updated + pushed automatically when we update directly on the _erc725.js_ repo?

Reply:
yes